### PR TITLE
#708: Maintenance update of libraries and code

### PR DESCRIPTION
**Pull request by bonkey**
#708: Maintenance update of libraries and code

### CODE REVIEW

File: AtlasSDK/AtlasSDK/Utils/Keychain.swift [hunk 1]
```diff
@@ -13,14 +13,15 @@ struct Keychain {
 
     @discardableResult
     static func delete(key: String) -> Bool {
+        let status: OSStatus
```
Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/Utils/Keychain.swift]: Actually I was surprised that `defer` sees the variable created after it, now it makes more sense

**Comment by hani-ibrahim** [comment]
👍

File: AtlasSDK/AtlasSDK/Utils/Keychain.swift (hunk 1 above)
Comment (bonkey) [inline AtlasSDK/AtlasSDK/Utils/Keychain.swift]: Me either, but when it worked I kept it.

I'm just thinking (only philosophically, without checking reasoning or internal mechanisms) if it did work, why change? Don't they go [too much into restrictions](http://blog.cleancoder.com/uncle-bob/2017/01/11/TheDarkPath.html)? 👹

**Comment by bonkey** [comment]
👍

**Comment by bonkey** [comment]
Merging without codecov – CI didn't fail, but don't need to play with it any longer.

**Pull request merged**